include.js

```diff
@@ -69,34 +69,6 @@ let cards = [
     author: 'Ibaad',
     githubLink: 'https://github.com/ibaaddurrani'
   },
-  {
-    artName: 'RichardsAnimation',
-    pageLink: './Art/Richard/animated.html',
-    imageLink: './Art/Richard/bounce.gif.gif',
-    author: 'Richard',
-    githubLink: 'https://github.com/richardhartleydev'
-  },
-  {
-    artName: 'Loader Animation',
-    pageLink: './Art/Abhishek/animation.html',
-    imageLink: './Art/Abhishek/animation.gif',
-    author: 'Abhishek Kumar',
-    githubLink: 'https://github.com/abhishekl1289'
-  },
-  {
-    artName: 'Fan',
-    pageLink: './Art/Samriddhi/fan.html',
-    imageLink: './Art/Samriddhi/fan.png',
-    author: 'Samriddhi',
-    githubLink: 'https://github.com/NarayanSam'
-  },
-  {
-    artName: 'RGB Square',
-    pageLink: './Art/TCrypt/animated.html',
-    imageLink: './Art/TCrypt/rgbsq-animated.gif',
-    author: 'T-Crypt',
-    githubLink: 'https://github.com/T-Crypt'
-  },
```

Comment:
Is there a reason why you are removing these entries?

Reply:
No ma'am, I was trying to fix the issue of  "fix conflicts"

Reply:
I understand. Please fix this issue so that you don't delete any valid entries in the `include.js` file.

Reply:
I added removed entries

Reply:
@amanr-dev there are now any changes to `include.js`, so your entry for this PR is also missing.